### ISSUE TITLE
Dark mode

### ISSUE THREAD

**Issue by JeanClaude2** [horosproject/horos#401]
The dark mode is amazing in mac os x mojave! I want it in Horos but I don't find where to activate it. It works well in Osirix, and I read somewhere that it also works in Horlix...

How to turn dark mode on in Horos?

**Comment by fvpolpeta** [comment]
Hi there -

So far, this function was not available in Horos. Now that it's natively part of Mojave, we will be working to integrating it into Horos. 

No time frame defined yet, but some moment between the end of this year and beginning of the next when version 4.0 will be releases.

Thanks for contact

Fauze & Horos Team

**Comment by JeanClaude2** [comment]
Great ! Will this be included in the free version of Horlix ?

**Comment by fvpolpeta** [comment]
Maybe.

HorosProject is an open source project sponsored by Purview and donations from the community.

HorliX has been making use of the Horos code basis and work to create their own software distribution, which also features a commercial version. In principle, that’s ok considering the license terms that rules Horos project (LGPL).

But on behalf of the open source initiative and with the goal of a free medical image viewer in mind, it would be great to have HorliX maintainers pushing back fixes/enhancements to Horos. This would be great for everyone, but so far it mostly has been an one way collaboration. It’s up to them. 

Fauze

**Comment by aglv** [comment]
My #412 include dark mode fixes that make Horos look good for me. If you notice any missing adjustments, please screenshot them and append to this thread.

**Comment by JeanClaude2** [comment]
Yes ! Thank you Purview team !

**Comment by aglv** [comment]
Now now, I don't work for Purview, and Horlix is not a Purview product. Just clarifying..

**Comment by JeanClaude2** [comment]
OK, so you didn't get any of the money donated to horosproject.org ?

**Comment by aglv** [comment]
Not a cent. But it's ok, their work is great, and I take advantage from it through other means.

**Comment by Hiroaki-Inomata** [comment]
I am the author of HorliX.

We offered the original code of ROI-color-rotation-UI and gave the suggestion for solving Philips US problem. Of cource, we have NOT got any of money from horosproject.org and/or purVIEW. 
I think it is not one way collaboration.

Horos project puts emphasis on open-source mind. I respect it, too.
But we, medical doctors, don't care so much whether the DICOM Viewer is open source or not when we check patients medical images like CR, CT, MRI...etc.


H. Inomata MD
PHAZOR, LLC

**Comment by fvpolpeta** [comment]
I believe this is not the channel for this kind of discussion. But I would like to add some considerations as some comments are not aligned to the essence of this project and what it means for Purview.

----------------------------------------------------------------------------------------

1) The collaboration made by @air-h-128k-il (HorliX member) (a) was implemented on top of a concept that already existed in OsiriX (b) suggested by @omrd (c) implemented on top of Horos existing code (d) improved by @brizolara in terms of efficiency.

If this is not a two-way collaboration, or more than that, a community-based collaboration I don't know what it is.

----------------------------------------------------------------------------------------

2) In respect to Phillips US fix, we were aware of it and just fixed. @Hiroaki-Inomata, you are not the only to report a bug in Horos. 

----------------------------------------------------------------------------------------

3) Most of the donations Purview received so far are (a) to keep the project alive in terms of marketing and infrastructure (b) and most recently to pursue the 510K FDA clearance, which is a long-term and very expensive process. To be transparent, so far Purview invested more than it made directly from Horos. And one could ask: "So why Purview puts its money and time in Horos?". The answer:

When OsiriX became a closed source project, and with the free version imposing limitations, some Purview clients that relied on OsiriX came to Purview to express how that affected their daily practice. Purview decided to find ways to support these users. That's when Horos emerged. Ultimately, Horos also served as a marketing vehicle to Purview solutions. 

With that said, and as @aglv stated, HorliX is not a Purview project, and Horos is an open source project sponsored by Purview. With that sponsorship it became possible to add some governance and maintainability to Horos. But Purview **already existed** before Horos with its own portfolio of solutions. Only now, in 2018, Purview decided to integrate some of its solutions into Horos by means of the Horos Cloud plugin. Solutions that were already there, as part of other projects that the company runs.

----------------------------------------------------------------------------------------

4) We really appreciate all the contributions we received. As a developer I'm personally thankful to @brizolara @aglv @jaasantinha and other engineers that contributed to the project on behalf of an open source and free medical imaging viewer. I'm not against HorliX - it's aligned to what the LGPL license states - but as an open source and free software advocate I would like to see they contributing back to Horos, rather than only grabbing code and promoting their own viewer in channels that were created exclusive to the Horos project. 

Last, to corroborate my line of thought, something to reason about:

There is in Horos more than a decade of work from the free and open source version of OsiriX, as there is in HorliX more than 3 years of work in Horos. Horos is free and open source and being so is the way I  consider fair to respect what was accomplished by the OsiriX initiative, and what this free viewer represents to medical practitioners, researchers, and professionals around the world.

Sincerely 

Fauze

**Comment by FabriceHx** [comment]
@fvpolpeta Sorry, but I did not understand anything…

We just want a free DICOM viewer, without bugs ! We don’t care if it’s name is Horlox or Horos, or who is the author of this feature.

**Comment by docftc** [comment]
@FabriceHx  you want a high level product for free?! complaining if something isn't working like in a $$$ software. You should be thankful that people make such an effort to keep software like that alive an develop it further!
If they try certain things which makes it unstable you can report it, but you can't expect it running like commercial SW. 

If have donated to Horos, because I use it on daily bases and I think it is more than fair to do so ... I had issues, too --> on one of my Macs I use "Horlix" (which runs stable) because Horos crashes with certain studies ... I am happy to pay a certain amount for a product if it is stable.
Once again, no offence but HOROS IS FOR FREE, so do not complain, you are free to use alternatives like OsiriX or HorliX.

**Comment by FabriceHx** [comment]
@docftc I fully agree with you : in fact, for now, we are using OsiriX MD… very stable and fast. I hoped to switch to Horos… I’ll try HoriX.

No offence, but free doesn’t always mean unstable or bugged. For example, Chrome is free, and it’s clearly the best Internet browser.

I’m not a developer, I’m a radiologist. I cannot help. As Dr H. Inomata said, most of us want to see CT & MR. We don’t really care about open-source or who is distributing/developing the software.

**Comment by Hiroaki-Inomata** [comment]
I think free software is different from freeware.
Freeware is free of charge.
"Free software is software that gives you the user the freedom to share, study and modify it. We call this free software because the user is free."
See: What is Free software? available at: https://www.fsf.org/about/what-is-free-software

In this meaning, HorliX is free software.

H. Inomata MD
PHAZOR, LLC

**Comment by world2005** [comment]
Hi,

I wonder how to disable the dark mode in v4.0. With the dark mode, it is hard to see DICOM metadata. Thanks.
